service/src/main/java/uk/gov/hmcts/reform/fpl/service/HearingBookingService.java

```diff
@@ -32,22 +33,14 @@
         return hearingDetails.stream().filter(this::isPastHearing).collect(toList());
     }
 
-    private boolean isPastHearing(Element<HearingBooking> element) {
-        return ofNullable(element.getValue())
-            .map(HearingBooking::getStartDate)
-            .filter(hearingDate -> hearingDate.isBefore(now()))
-            .isPresent();
-    }
-
-    // TODO: this method will always get the first (by date, even if in past) hearing booking. Not the most urgent
-    // FPLA-1484
     public HearingBooking getMostUrgentHearingBooking(List<Element<HearingBooking>> hearingDetails) {
         if (hearingDetails == null) {
             throw new IllegalStateException("Hearing booking was not present");
         }
 
         return hearingDetails.stream()
             .map(Element::getValue)
+            .filter(e -> !e.getStartDate().toLocalDate().isBefore(LocalDate.now()))
```

Comment:
we can use isAfter to remove the !

Reply:
I also would prefer hearing rather than e

Reply:
Yup, I will change.

Reply:
> we can use isAfter to remove the !
I had to keep it as is because isAfter excludes today's date.